From: Brendan William <brendan.william@outlook.com>
Date: Fri, 4 Aug 2023 22:44:15 -0700
Subject: style: Use privacy_indicator_color in overview for screen sharing
 indicator

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2859>
(cherry picked from commit b0ed63083b267eaf1f858087a5aa8ae2b6a3b55c)
---
 data/theme/gnome-shell-sass/widgets/_panel.scss | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/theme/gnome-shell-sass/widgets/_panel.scss b/data/theme/gnome-shell-sass/widgets/_panel.scss
index f23aa4b..f19c7c7 100644
--- a/data/theme/gnome-shell-sass/widgets/_panel.scss
+++ b/data/theme/gnome-shell-sass/widgets/_panel.scss
@@ -116,7 +116,7 @@ $panel_transition_duration: 250ms; // same as the overview transition duration
       }
 
       &.screen-sharing-indicator {
-        @include panel_button($warning_color, transparentize(black, 0.2), $flat: false);
+        @include panel_button($privacy_indicator_color, $flat: false);
       }
     }
   }
